src/DataFixtures/TaxonomyFixtures.php

```diff
@@ -41,45 +41,19 @@ private function loadTaxonomies(ObjectManager $manager): void
         foreach ($this->config as $taxonomyDefinition) {
             if (! empty($taxonomyDefinition['options'])) {
                 $options = $taxonomyDefinition['options'];
-            } else {
-                $options = $this->getOptions();
-            }
-
-            foreach ($options as $key => $value) {
-                if (is_numeric($key)) {
-                    $key = $value;
+                foreach ($options as $key => $value) {
+                    $taxonomy = Taxonomy::factory(
+                        $taxonomyDefinition['slug'],
+                        $key,
+                        $value,
+                        $taxonomyDefinition['has_sortorder'] ? $order++ : 0
+                    );
+
+                    $manager->persist($taxonomy);
+                    $reference = 'taxonomy_' . $taxonomyDefinition['slug'] . '_' . Str::slug($key);
+                    $this->addReference($reference, $taxonomy);
                 }
-
-                $taxonomy = Taxonomy::factory(
-                    $taxonomyDefinition['slug'],
-                    $key,
-                    $value,
-                    $taxonomyDefinition['has_sortorder'] ? $order++ : 0
-                );
-
-                $manager->persist($taxonomy);
-                $reference = 'taxonomy_' . $taxonomyDefinition['slug'] . '_' . Str::slug($key);
-                $this->addReference($reference, $taxonomy);
             }
         }
     }
-
-    private function getOptions()
```

Comment:
Why remove these?

Reply:
are they needed for something?

Reply:
I think it makes sense for the dummy content "fixtures" to have a bunch of tags. Later on, it'll help when we need to add tests for dealing with tags too. So, i'd prefer to keep them.

Reply:
@JarJak True, but during development, it's useful to just have a "bunch of tags" available. Can we add a flag to the fixtures code, that allows toggling it?